### PR TITLE
Tests for editableList

### PR DESCRIPTION
**Pull request by albarivas**
### What does this PR do?
Adds tests for editableList

### What issues does this PR fix or reference?

#<Insert GitHub Issue>

## The PR fulfills these requirements:

[x] Tests for the proposed changes have been added/updated.
[x] Code linting and formatting was performed.

### Functionality Before

editableList didn't have tests

### Functionality After

editableList has tests

### CODE REVIEW

File: force-app/main/default/lwc/editableList/__tests__/editableList.test.js [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+const getAccountsAdapter = registerApexTestWireAdapter(getAccounts);
+
+describe('c-editable-list', () => {
+    afterEach(() => {
+        // The jsdom instance is shared across test cases in a single file so reset the DOM
+        while (document.body.firstChild) {
+            document.body.removeChild(document.body.firstChild);
+        }
+    });
+    it('renders lightning-datatable when there is data', () => {
```
Comment (pozil) [inline force-app/main/default/lwc/editableList/__tests__/editableList.test.js:50]: Add line break

File: force-app/main/default/lwc/editableList/__tests__/editableList.test.js [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+
+        // Emit data from @wire
+        getAccountsAdapter.emit(mockGetAccounts);
+
+        // Return a promise to wait for any asynchronous DOM updates. Jest
+        // will automatically wait for the Promise chain to complete before
+        // ending the test and fail the test if the promise rejects.
+        return Promise.resolve().then(() => {
+            const datatableEl = element.shadowRoot.querySelector(
+                'div.table-container > lightning-datatable'
```
Comment (pozil) [inline force-app/main/default/lwc/editableList/__tests__/editableList.test.js:65]: Simplify that selector, there's only one datatable that will ever be rendered here.

File: force-app/main/default/lwc/editableList/__tests__/editableList.test.js [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+
+        // Return a promise to wait for any asynchronous DOM updates. Jest
+        // will automatically wait for the Promise chain to complete before
+        // ending the test and fail the test if the promise rejects.
+        return Promise.resolve().then(() => {
+            const errorPanelEl = element.shadowRoot.querySelector(
+                'c-error-panel'
+            );
+            expect(errorPanelEl).not.toBeNull();
+            expect(errorPanelEl.errors).toBeTruthy();
```
Comment (pozil) [inline force-app/main/default/lwc/editableList/__tests__/editableList.test.js:91]: No need to test that: it's already covered above (error panel wouldn't show up if it was falsy).

File: force-app/main/default/lwc/editableList/__tests__/editableList.test.js (hunk 1 above)
Comment (albarivas) [reply force-app/main/default/lwc/editableList/__tests__/editableList.test.js:50]: done

File: force-app/main/default/lwc/editableList/__tests__/editableList.test.js (hunk 3 above)
Comment (albarivas) [reply force-app/main/default/lwc/editableList/__tests__/editableList.test.js:91]: What I am testing is that we pass in the errors property in the parent to the errors property in errorPanel, it's an additional check.

File: force-app/main/default/lwc/editableList/__tests__/editableList.test.js (hunk 2 above)
Comment (muenzpraeger) [reply force-app/main/default/lwc/editableList/__tests__/editableList.test.js]: `element.shadowRoot.querySelector('lightning-datatable');`

File: force-app/main/default/lwc/editableList/__tests__/editableList.test.js (hunk 3 above)
Comment (pozil) [reply force-app/main/default/lwc/editableList/__tests__/editableList.test.js:92]: Can you test the value or `errors` instead of just truthy?

**Pull request merged**